lab-yunjoo/test/storage-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Storage = require('../lib/storage');
+const testStorage = new Storage(`${__dirname}/data`);
+const fs = require('fs');
+
+describe('testing storage module', function(){
+  describe('testing setItem', function(){
+    it('should return the item', function(done){
+      testStorage.setItem('note', {'id': '123456', 'content':'test test test!'})
```

Comment:
It's considered best practice not to wrap object keys in quotes unless you need to. So, this should be written as 
```javascript
{ id: '123456', content:'test test test!' }
```
But if there is a case where you need to wrap a key in quotes, wrap only that key in quotes. For example:
```javascript
{ id: '123456', 'content-type':'test test test!' }
```